### DISCUSSION TITLE
Mini Dexed - 8 part live sequencing, no EQ or any audio processing

### DISCUSSION THREAD

**Discussion by izash** [probonopd/MiniDexed#231] [Show and tell]
[https://fb.watch/d2rHCvczfA/](https://fb.watch/d2rHCvczfA/)

**Comment by probonopd** [comment]
Wow, this is very cool! Congrats. :+1:

**Reply by izash** [reply]
Thanks to your great work!

**Reply by sou7611** [reply]
Are you getting 8 discrete audio signals out?

That's what it looks like on your sequencer (and what software is that?)

**Reply by izash** [reply]
What you see on my sequencer (Avid Protools) are 8 midi tracks. 
Mini Dexed currently only allows 2 discreet outputs.

**Comment by sou7611** [comment]
Aha, gotcha.

Now that springs an interesting question.

HDMI supports up to 8 PCM audio tracks...

🤔

**Reply by probonopd** [reply]
The only question would be: how do you get them out of there. And: Would any (semi-)pro touch HDMI, due to its latency?

**Comment by sou7611** [comment]
There are many examples of audio de-embedders that can extract the audio.

HDMI is used in the pro world, essentially it's interchangeable with SDI
across short distances.

The BlackMagic Decklink and UltraStudio interfaces can also drop tracks
directly into DAWs off their HDMI or SDI inputs.

It's an intriguing idea, if you truly want 8 separate channels you can then
process/mix/whatnot.

**Reply by izash** [reply]
Remember that this is a musical instrument played in real time, not the audio outputs of a recorded video. 
Any latency above 2-3ms would be prohibitive.

**Comment by sou7611** [comment]
It would be the same latency as using HDMI for stereo audio.

De-embedders do not add latency, typically.

I'd expect import into a DAW to be even faster as there no DAC stage
involved.
